pkg/api/types.go

```diff
@@ -513,8 +523,8 @@ type PodSpec struct {
 // PodStatus represents information about the status of a pod. Status may trail the actual
 // state of a system.
 type PodStatus struct {
-	Phase PodPhase `json:"phase,omitempty"`
-
+	Phase        PodPhase        `json:"phase,omitempty"`
+	ServicePhase PodServicePhase `json:"servicePhase,omitempty"`
```

Comment:
Makes sense. PodCondition will start with on condition which will be PodInService.